.github/workflows/tag-publish.yml

```diff
@@ -130,20 +89,28 @@ jobs:
         restore-keys: v1-${{ runner.os }}-gradle-${{ hashfiles('**/gradle-wrapper.properties') }}
     - name: Grant execute permission for gradlew
       run: chmod +x gradlew
+    - name: "Publish to Artifactory"
+      run: ./gradlew artifactoryPublish
+      env:
+        ARTIFACTORY_USERNAME: ${{ secrets.ARTIFACTORY_USERNAME }}
+        ARTIFACTORY_PASSWORD: ${{ secrets.ARTIFACTORY_PASSWORD }}
+        ARTIFACTORY_REPO_KEY: "libs-snapshot-local"
+    - name: Auth to GCR
+      uses: google-github-actions/setup-gcloud@master
+      with:
+        version: '270.0.0'
+        service_account_email: ${{ secrets.GCR_PUBLISH_EMAIL }}
+        service_account_key: ${{ secrets.GCR_PUBLISH_KEY }}
+    - name: Explicitly auth Docker for GCR
+      run: gcloud auth configure-docker --quiet
     - name: Construct docker image name and tag
       id: image-name
       run: |
-        DOCKER_TAG=""
-        if ${{github.event_name == 'pull_request' }}; then
-          DOCKER_TAG=pr${{ github.event.number }}-$(echo ${{ github.event.pull_request.head.sha }} | cut -c1-7)
-        elif ${{github.event_name == 'push' }}; then
-          DOCKER_TAG=${{ steps.tag.outputs.tag }}
-        fi
+        DOCKER_TAG=${{ steps.tag.outputs.tag }}
         echo ::set-output name=name::gcr.io/${GOOGLE_PROJECT}/${SERVICE_NAME}:${DOCKER_TAG}
     - name: Build image locally with jib
       run: "./gradlew jibDockerBuild --image=${{ steps.image-name.outputs.name }} -Djib.console=plain"
     - name: Run Trivy vulnerability scanner
-      # https://github.com/broadinstitute/dsp-appsec-trivy-action
```

Comment:
Is there a reason to remove this convenience link to the source of this action?

Reply:
No. I thought it was old code. I'll put it back with more context.